envoy/metadata.csv

```diff
@@ -15,6 +15,7 @@ envoy.cluster_manager.cds.update_attempt,count,,request,,Total API fetches attem
 envoy.cluster_manager.cds.update_success,count,,request,,Total API fetches completed successfully,1,envoy,cds successful api accesses
 envoy.cluster_manager.cds.update_failure,count,,request,,Total API fetches that failed because of network errors,-1,envoy,cds failed api accesses
 envoy.cluster_manager.cds.update_rejected,count,,request,,Total API fetches that failed because of schema/validation errors,-1,envoy,cds rejected api accesses
+envoy.cluster_manager.cds.update_time,gauge,,millisecond,,Timestamp of the last successful API fetch attempt as milliseconds since the epoch,0,envoy,cds time api access
```

Comment:
Can we add in the description of new metrics that is for v3+ only?

Reply:
I assume the rest were just missing?

Reply:
Yes, the other metrics are available in v2